packages/financial-templates-lib/src/logger/Transports.ts

```diff
@@ -77,6 +80,15 @@ export function createTransports(transportsConfig: TransportsConfig = {}): Trans
         )
       );
     }
+    // If there is a Pagerduty V2 API key then add the pagerduty winston transport.
+    if (transportsConfig.pagerDutyV2Config || process.env.PAGER_DUTY_V2_CONFIG) {
```

Comment:
general question: if we have the v2 transport, is the intention to eventually move away all the v1 configs to the v2 config? I get that the structure is slightly different so we'll need to be sure to update all the bot configs accordingly but it seems somewhat redundant to support the old one long term. wydt?

Reply:
i think it will be safe to remove v1 in the medium term, would just like to be confident that this v2 solution is working as needed before that happens